spec/lib/sendgrid_helper_spec.rb

```diff
@@ -61,4 +61,41 @@
       end
     end
   end
+
+  context "sendgrid connection test" do
+    let :host do
+      'smtp.sendgrid.net'
+    end
+
+    let :port do
+      587
+    end
+
+    context "when it connects" do
+      it "will return true" do 
+        SendgridHelper.smtp_alive?(host, port).should be_true
```

Comment:
Does this test hit the actual SMTP server? That's probably bad form to have our tests open a socket to someone elses mail server every time we run tests.